ui/hooks/gasFeeInput/useMaxFeePerGasInput.js

```diff
@@ -117,7 +131,7 @@ export function useMaxFeePerGasInput({
       gasFeeEstimates,
       gasEstimateType,
       estimateToUse,
-      maxFeePerGasFromTransaction,
+      initialMaxFeePerGas || 0,
```

Comment:
and then this could just be `initialMaxFeePerGas` without `|| 0`

Reply:
plz check above